src/main/java/org/mule/module/xml/internal/operation/XsltOperation.java

```diff
@@ -81,6 +83,14 @@ public String xsltTransform(@Content(primary = true) InputStream content,
     return withTransformer(xslt, transformer -> {
       bindParameters(transformer, contextProperties);
 
+      // TODO: Question: Should I put this in try-finally block to deal with the case something fails in between,
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS1135))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-core-extensions-mule-xml-module&pullRequest=29&issues=AWilgRIQotIXnmsUpHVk&open=AWilgRIQotIXnmsUpHVk)

[ISSUE_KEY]: AWilgRIQotIXnmsUpHVk